cmake/Codegen.cmake

```diff
@@ -46,10 +46,11 @@ endfunction(GEN_BACKEND)
 
 set(RegisterXPU_PATH ${BUILD_TORCH_XPU_ATEN_GENERATED}/RegisterXPU.cpp)
 set(XPUFallback_PATH ${TORCH_XPU_OPS_ROOT}/src/ATen/native/xpu/XPUFallback.template)
+set(XPU_AOTI_INSTALL_DIR ${TORCH_ROOT}/torch/csrc/inductor/aoti_torch/generated/extend)
```

Comment:
Can we use PyTorch defined CMAKE var for the path?

Reply:
@fengyuan14 there is no available CMAKE var for the path.